[IMP] spreadsheet: remove default templates (#11832)
odoo/documentation: content/applications/productivity/spreadsheet/templates.rst

Templates
=========

> In *Documents*, any spreadsheet can be saved as a template. This lets you quickly create

Reviewer: ```suggestion
In Documents, any spreadsheet can be saved as a template, allowing you to quickly create
```
- Italics aren't necessary for app names
- I would group the two sentences into one since they're quite simple.

spreadsheets in the future without starting from scratch. For example, you could create a template
for a monthly budget report or a quarterly sales commission report.

> .. _templates/create:

Reviewer: ```suggestion
.. _spreadsheet/templates/create:
```
We try to use the following syntax for anchor names: app-name/file-name/custom-anchor ( https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#custom-anchors)

Create a template
=================

> To create a template from an existing spreadsheet, open the spreadsheet, then click
> :menuselection:`File --> Save as template` from the menu bar. Modify the Template Name if necessary
> and click :guilabel:`Confirm`.

Reviewer: ```suggestion
To create a template from a spreadsheet, open or create the spreadsheet, then click
:menuselection:`File --> Save as template` from the menu bar. Modify the :guilabel:`Template Name` if necessary and click :guilabel:`Confirm`.
```
- To also mention the fact that people could create a new spreadsheet to use it as a template?
- I would also add a custom anchor linking to this section for creating spreadsheets: https://www.odoo.com/documentation/master/applications/productivity/documents.html#spreadsheet
- We also use the `guilabel`markup for field names (or any UI element): https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#gui-element

Author: Yes for points 1 and 3. For point 2, I would like to have https://www.odoo.com/documentation/master/applications/productivity/documents.html#spreadsheet updated slightly ... is that MASP or do I ask her if I can do it myself?

Templates
=========

Spreadsheet templates allow you to quickly create spreadsheets without starting from scratch. For
example, you could create a template for a monthly budget report or a quarterly sales commission
report.

.. note::
   Templates are available to all users on the database.

.. _spreadsheet/templates/create:

Create a template
=================

Any spreadsheet can be saved as a template. Open the relevant spreadsheet or :ref:`create a new one
<documents/documents/spreadsheet>`. From the menu bar, click :menuselection:`File -->
Save as template`. Modify the default :guilabel:`Template Name` if needed and click
:guilabel:`Confirm`.

.. important::
   Once a spreadsheet is saved as a template, any further changes to the open spreadsheet are
   saved only in that spreadsheet. To make changes to the template, you need to :ref:`edit
   the template <spreadsheet/templates/manage>`.

You can also create a template by :ref:`making a copy of an existing template and editing it
<spreadsheet/templates/manage>`.

.. _spreadsheet/templates/use:

Use templates
=============

> To create a new spreadsheet using a template, click :menuselection:`New --> Spreadsheet`.
> Select the relevant template and click :guilabel:`Create`.

Reviewer: ```suggestion
To create a new spreadsheet using a template, open the Documents app and click :menuselection:`New --> Spreadsheet`.
Select the relevant template and click :guilabel:`Create`.
```
I think I would mention the app name as well here to provide a bit more context and "starting point" if that makes sense?

Alternatively, go to :menuselection:`Documents --> Configuration --> Spreadsheet Templates` then,
on the line of the relevant template, click :icon:`fa-plus` :guilabel:`New spreadsheet`.

.. note::

>    By default, the new spreadsheet inherits the name of the template; this is shown at the top of
>    the screen. Select the name then rename the spreadsheet.

Reviewer: ```suggestion
   By default, the new spreadsheet inherits the name of the template, which is shown at the top of
   the screen. To rename the spreadsheet, click the name and edit it.
```
Probably a bit nitpicky but I want to try and get rid of "this" unless there's no other solution and clarify the second sentence a little bit by starting with the intention.

Remove the :guilabel:`My Templates` :ref:`filter <search/preconfigured-filters>` to view all
templates in the database.

> To:

Reviewer: This might be preferential but I think I would repeat the "to" instead of this? Or even reword the introductory sentence and list items to avoir repeating "next to the desired template"?

> - edit an existing template, click the :icon:`fa-pencil` icon next to the desired template.
> - make a copy of an existing template, click the :icon:`fa-clone` icon next to the desired
>   template.

Reviewer: ```suggestion
- edit an existing template, click :icon:`fa-pencil` :guilabel:`Edit` next to the desired template.
- make a copy of an existing template, click :icon:`fa-clone` :guilabel:`Make a copy` next to the desired
  template.
```
Shoud we also mention + New Spreadsheet?

Author: In the end, in light of your comments (about + New Spreadsheet, and about creating a new spreadsheet and then saving as a template, and with a further step back, I want to make bigger changes to this page. So the next version you see will be a little different.

Modifications are automatically saved.

.. tip::

>    Use the download button under the :guilabel:`Spreadsheet file` column to export a template in
>    JSON format. The file can be imported into another database.

Reviewer: ```suggestion
   Use the :icon:`fa-download` (:guilabel:`download`) button in the :guilabel:`Spreadsheet file` column to export a template in JSON format. The file can then be imported into another database.
```

Remove the :guilabel:`My Templates` :ref:`filter <search/preconfigured-filters>` to view all
templates in the database.

Various actions are possible:

- Copy an existing template by clicking :icon:`fa-clone` :guilabel:`Make a copy` at the right of

>   the screen. The new template appears at the bottom of the list. Click on the :guilabel:`Name` and
>   start typing to rename the template.

Reviewer: ```suggestion
  the screen. The new template appears at the bottom of the list. Click the :guilabel:`Name` to edit it.
```
also a bit nitpicky :grimacing:

- Edit an existing template (including one you just copied) by clicking :icon:`fa-pencil`

>   :guilabel:`Edit`, then make the required changes. Modifications are

Reviewer: ```suggestion
  :guilabel:`Edit`, then making the required changes. Modifications are
```

  automatically saved.

> - Delete a template by ticking the checkbox next to it, then, under the :icon:`fa-cog`
>   :guilabel:`Actions` menu, click :guilabel:`Delete`.

Reviewer: ```suggestion
- Delete a template by ticking the checkbox next to it, clicking :icon:`fa-cog` :guilabel:`Actions`, then :guilabel:`Delete`.
```
